### PR TITLE
Adding axe, axe-beta and axe-con

### PR DESCRIPTION
**Pull request by ljgoslin**
First time adding changes to this doc, hopefully, I did it right - thanks all!

### CODE REVIEW

File: src/_data/resources.json [hunk 1]
```diff
@@ -26,6 +26,11 @@
 			"description": "24 days of digital accessibility gifts during the season of giving and sharing.",
 			"url": "https://www.24a11y.com/"
 		},
+		{
+			"title": "Deque Blog",
```
Comment (ericwbailey) [inline src/_data/resources.json:30]: Can you move this into alphabetical order? It would be after "Blog - Karl Groves", starting on line 69.

File: src/_data/resources.json [hunk 2]
```diff
@@ -143,6 +148,11 @@
 			"description": "Building an elegant, functional website requires more than just knowing how to code. In Adaptive Web Design, Second Edition, you’ll learn how to use progressive enhancement to build websites that won’t break, work anywhere, are accessible by anyone—on any device, and are designed to work well into the future.",
 			"additional": "Aaron Gustafson",
 			"url": "http://a.co/d/7aVobuX"
+		},
+				{
+			"title": "Agile Accessibility Handbook",
+			"description": "Free ebook: A Practical Guide to Accessible Software Development at Scale": "Dylan Barrell",
```
Comment (ericwbailey) [inline src/_data/resources.json:154]: It looks like the nested quotes are breaking the build. Let's try this instead:


`"description": "Free ebook: A Practical Guide to Accessible Software Development at Scale, by Dylan Barrell.",`

File: src/_data/resources.json [hunk 3]
```diff
@@ -265,6 +275,27 @@
 			"browsers": [
 				"Chrome"
 			]
+		},
+				{
+			"title": "axe",
+			"description": "Chosen by Microsoft, Google, development and testing teams everywhere, axe is the World’s leading digital accessibility toolkit.",
```
Comment (ericwbailey) [inline src/_data/resources.json:281]: Can we rephrase this to speak to what axe does, not who uses it? Right now it's unclear.

File: src/_data/resources.json [hunk 4]
```diff
@@ -265,6 +275,27 @@
 			"browsers": [
 				"Chrome"
 			]
+		},
+				{
```
Comment (ericwbailey) [inline src/_data/resources.json:279]: Looks like the indentation here is off.

File: src/_data/resources.json [hunk 5]
```diff
@@ -265,6 +275,27 @@
...
+			"additional": "Deque",
+			"url": "https://www.deque.com/axe/",
+			"browsers": [
+				"Chrome",
+				"Firefox",
+				"Edge",
+			]
+		},
+				},
+				{
```
Comment (ericwbailey) [inline src/_data/resources.json:291]: Indentation is off here, as well.

File: src/_data/resources.json [hunk 6]
```diff
@@ -806,6 +837,12 @@
 			"additional": "Michigan, United States",
 			"url": "https://twitter.com/annarbora11y"
 		},
+		{
+			"title": "axe-con",
+			"description": "free, virtual digital accessibility conference with tracks for design, develoopers, business leaders, and compliance",
```
Comment (ericwbailey) [inline src/_data/resources.json:842]: Can we capitalize the first word and add a period after the word "compliance"? We try to use full sentences for our descriptions.

**Review by ericwbailey — CHANGES_REQUESTED** [review]

**Comment by ljgoslin** [comment]
@ericwbailey I've made the changes - please review again. Thanks!

File: src/_data/resources.json [hunk 7]
```diff
@@ -266,6 +276,26 @@
...
 		},
+		{
+			"title": "axe",
+			"description": "The world's leading digital accessibility toolkit - download as a browser extenion or on NPM. Chosen by Microsoft, Google, development and testing teams everywhere.",
+			"additional": "Deque",
+			"url": "https://www.deque.com/axe/",
+			"browsers": [
+				"Chrome",
+				"Firefox",
+				"Edge",
```
Comment (ericwbailey) [inline src/_data/resources.json:287]: Remove this trailing comma.

File: src/_data/resources.json [hunk 8]
```diff
@@ -266,6 +276,26 @@
...
+			]
+		},
+		{
+			"title": "axe beta",
+			"description": "Automatically catch tons of accessibility bugs with axe. Catch even more with axe beta.",
+			"additional": "Deque",
+			"url": "https://www.deque.com/axe/beta/",
+			"browsers": [
+				"Chrome",
```
Comment (ericwbailey) [inline src/_data/resources.json:296]: Remove this trailing comma.

**Review by ericwbailey — CHANGES_REQUESTED** [review]
@ljgoslin Almost there! The last, really annoying thing about JSON is if a comma is present for the last item in a list it'll break things.

**Comment by ljgoslin** [comment]
Ah gotcha, removed! I think we're good now - danke :)

**Review by ericwbailey — APPROVED** [review]
Woohoo! Thank you!
